test/v2/controllers/CreateAmendForeignPropertyAnnualSubmissionControllerSpec.scala

```diff
@@ -125,108 +66,90 @@ class CreateAmendForeignPropertyAnnualSubmissionControllerSpec
           .wrap((), CreateAmendForeignPropertyAnnualSubmissionHateoasData(nino, businessId, taxYear))
```

Comment:
Would be good to pull the hateoas data out into a val that we can add to the test trait with the other data

`
MockHateoasFactory
          .wrap((),hateoasData)
`

`
 protected val hateoasData: CreateAmendForeignPropertyAnnualSubmissionHateoasData = CreateAmendForeignPropertyAnnualSubmissionHateoasData(nino, businessId, taxYear)
`